KS-16-01-playing-dice

DenisGubernatorov opened the pull request:
1) Класс PlayingDice с приватным полем, где генерируется результат броска кубика от 1 до 6. Реализован публичный метод для вывода результата броска

DenisGubernatorov committed:
KS-16-01-playing-dice
1) Класс PlayingDice с приватным полем, где генерируется результат броска кубика от 1 до 6. Реализован публичный метод для вывода результата броска

ievetrovdev commented on src/main/kotlin/lesson16/lesson16_task1.kt:
```diff
@@ -1 +1,16 @@
+import kotlin.random.Random
+
+class PlayingDice {
+    private var diceRollResult = Random.nextInt(1, 7)
```
Любой импорт – лишний код в классе. Неиспользуемый импорт – неиспользуемый код. Существуют условно тонкие материи, когда надо использовать какую-то конкретную реализацию под капотом. Поэтому обоснованно может быть вызвать некую альтернативу из Котлин или похожий функционал, но с другим внутренним алгоритмом из Джавы. Ситуативно. Здесь я бы использовал экстеншн получения рандомного числа из диапазона: (1..6).random()

DenisGubernatorov committed:
KS-16-01-playing-dice
Изменение получения случайного числа: замена Random.nextInt(1, 7) на функцию-расширение random для диапазона (1..6).

ievetrovdev reviewed (approved):
Все отлично, верное решение!
